cdap-hbase-compat-base/src/main/java/co/cask/cdap/data2/transaction/coprocessor/CConfigurationCache.java

```diff
@@ -84,7 +109,14 @@ public void run() {
             try {
               CConfiguration newCConf = cConfReader.read();
               if (newCConf != null) {
+                Configuration newConf = new Configuration();
```

Comment:
This means we will create a new config object every 5 mins, no matter whether there is an update or not, right?